python/core/auto_generated/qgsauxiliarystorage.sip.in

```diff
@@ -338,7 +338,7 @@ Saves the current database.
 :return: ``True`` if everything is saved, ``False`` otherwise
 %End
 
-    QgsAuxiliaryLayer *createAuxiliaryLayer( const QgsField &field, QgsVectorLayer *layer ) const /Factory/;
+    QgsAuxiliaryLayer *createAuxiliaryLayer( const QgsField &field, QgsVectorLayer *layer, QString *errorMessage /Out/ = 0 ) const /Factory/;
```

Comment:
This is an API break -- Python callers will now receive a tuple return value instead of a QgsAuxiliaryLayer value

Reply:
Indeed. I proposed something different to avoid the API break.